contracts/cw1/src/lib.rs

```diff
@@ -97,6 +90,6 @@ mod tests {
 
     #[test]
     fn exec_msgs() {
-        assert_eq!(ExecMsg::messages(), ["AddMember", "RemoveMember"]);
+        assert_eq!(ExecMsg::messages(), ["AddMember"]);
```

Comment:
I didn't notice it before - you want it to be "add_member", not "AddMember" - it would be way useful, as it would be always used with incoming jsons. Check out the `convert_case` crate (already in dependencies).